packages/Webkul/Product/src/Http/Requests/ProductForm.php

```diff
@@ -66,6 +67,16 @@ public function rules()
             'special_price'      => ['nullable', new Decimal, 'lt:price'],
         ]);
 
+        if(isset(request()->images['files'])) {
```

Comment:
Add space after if

Reply:
We should request()->hasFile method instead of this

https://laravel.com/docs/5.5/requests#files